### PR TITLE
Vaishnavi_Assignment_35_15min_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
image

### CODE REVIEW

**Review by harshvegada — APPROVED** [review]
Correct Catch
Keep it up @VaishnaviVJun20

File: src/vaishnaviVJun20_Collection/Retrieve.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	void retrieveAllUniqueNames(ArrayList<String> name1, ArrayList<String> name2) {
+		name2.removeAll(name1);
+		name1.addAll(name2);
+		System.out.println("Output: " +name1);
+	}
+
+	public static void main(String[] args) {
+		String[] name1 = {"Palak", "Viresh", "Yash", "Aavruti"};
+		String[] name2 = {"Deavina","Palak","Viresh", "Nikhil"};
+		ArrayList<String> arrList1 = new ArrayList<String>(Arrays.asList(name1));
```
Comment (KrishnaKTechnocredits) [inline src/vaishnaviVJun20_Collection/Retrieve.java:23]: I am happy, you could figure out arrayList2.removeAll(arrayList1) is required and asList method returns List which could not be modified, hence we need to create new ArrayList object and call asList method from constructor.
Code looks good to me.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me. Keep it up vaishnavi.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me. Keep it up

**Pull request merged**